### PR TITLE
Add "Terra is coming" banner

### PR DESCRIPTION
**Pull request by davidangb**
DataBiosphere/firecloud-app#283

* removes the free trial banner system from the UI (keeps backend APIs untouched)
* removes free trial automated tests
* adds a "Terra is coming" banner, dismissable for the life of the current page

The svg files used for the banner visuals are copied from the terra-ui repo.

here's what it looks like,  against dev data:
FireShot Capture 056 - FireCloud - Broad Institute - local broadinstitute org

-----

- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by davidangb** [comment]
build failed with a GCR error; hopefully  transient. Jenkins  retest.

**Comment by davidangb** [comment]
had one good build, now another GCR error.  jenkins  retest.

**Comment by davidangb** [comment]
yet another GCR error. jenkins  retest.

File: src/cljs/main/broadfcui/common/notifications.cljs [hunk 1]
```diff
@@ -115,154 +115,56 @@
...
-                          {:text "Accept"
-                           :data-test-id "accept-terms-of-service"
-                           :disabled? (when-not (and terms-agreed? cloud-terms-agreed?)
-                                        "You must check the boxes to accept the agreement.")
-                           :onClick accept-eula})])}]))})
+    (fn [{:keys [state this]}]
+      (let [{:keys [dismissed?]} @state]
+        (when (not dismissed?)
+          [:div {}
+            [:div {:style {:height "66px"
```
Comment (zarsky-broad) [inline src/cljs/main/broadfcui/common/notifications.cljs:124]: Indentation looks a little funky, key on next line should be aligned with this key, not the brace. Also, `px` values should be able to be just specified as integers.

**Review by zarsky-broad — APPROVED** [review]
Minor comments on formatting, looks good generally. Didn't test.

**Pull request merged**